crates/biome_js_analyze/src/lint/style/use_import_type.rs

```diff
@@ -135,6 +145,12 @@ impl Rule for UseImportType {
                     AnyJsCombinedSpecifier::JsNamespaceImportSpecifier(namespace_specifier) => {
                         let namespace_binding = namespace_specifier.local_name().ok()?;
                         let namespace_binding = namespace_binding.as_js_identifier_binding()?;
+                        if ctx.has_jsx_runtime(JsxRuntime::ReactClassic)
```

Comment:
Why not directly return the value of the runtime?

```suggestion
                        if ctx.jsx_runtime() == JsxRuntime::ReactClassic
```

Reply:
No particular reason, really. I'll change it.